x-pack/legacy/plugins/apm/public/components/shared/Links/apm/ExternalLinks.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { IBasePath } from 'kibana/server';
+import url from 'url';
+
+export const getTraceUrl = (
+  kibanaBasePath: IBasePath,
+  traceId: number | string
+) =>
```

Comment:
Shouldn't this also take `start` and `end`?

Reply:
@sqren I will change it to `rangeFrom` and `rangeTo`, to make it consistent with what we have on the other routes.